fractions/javaee/bean-validation/README.adoc

```diff
@@ -0,0 +1,4 @@
+# Bean Validation
+
+The Bean Validation fraction provides support for JSR 303,
+providing class-level constraint and validation.
```

Comment:
I find this formulation unclear. I'd go with something like _Provides class-level constraint and validation according to JSR 303._